active/CornerRadius/CornerRadiusFilterConverter.md

```diff
@@ -0,0 +1,37 @@
+# Background
+
+Some controls need to turn on/off part of their rounded corners depending on their states.
+
+e.g. When CommandBarFlyout opens up its secondary menu, primary and secondary menus are clipped together we need to clear rounded corners on bottom of primary menu and top of secondary menu to close the gaps.
+
+![CommandBarFlyout CornerRadius example](images/commandbarflyout-cornerradius-example.png)
+
+You can't set individual values of a CornerRadius object in markup according to our [doc](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.CornerRadius#notes-on-xaml-syntax). We need a way to get non-uniform corner radius values to implement the design above.
+# Description
+
+CornerRadiusFilterConverter helps us filter CornerRadius values on specific directions and set values to 0 for the rest.

```

Comment:
Precedent for this kind of problem is CommandBarTemplateSettings. Why aren't se using that here?

Reply:
Yes, I think TemplateSettings works for CommandBarFlyout. There are other controls like AutoSuggestBox and ComboBox with similar problems but their code is not in MUX. We are trying to solve the problem without touching code in OS.